Controller/Views/CalendarController.php

```diff
@@ -26,16 +33,16 @@ public function updateAction(View $view, Request $request)
                 $field = $view->getContentType()->getFieldType()->__get('ems_'.$view->getOptions()['dateRangeField']);
                 
 
-                /**@var \DateTime $from */
-                /**@var \DateTime $to */
-                $from = new \DateTime($request->request->get('start', false));
+                /**@var DateTime $from */
+                /**@var DateTime $to */
```

Comment:
the @var definition should not be necessary when the type is the same as the created Class.
+ it won't work anyway if there is no space between /** @var